scripts/tasks/create-sidebar.ts

```diff
@@ -178,6 +178,22 @@ export const createSidebar = async (root: string, destination: string) => {
       prettierConfig
     );
 
+    for (const id of Object.keys(sidebars)) {
```

Comment:
issue(blocking): this won't work because the `formattedSidebarString` is parsed before we do any of the in-place sorting.

I also think we would want to only have this available for the `API` `Structures` categories, rather than every single sidebar category recursively.